cmd/do-agent/config.go

```diff
@@ -116,6 +117,9 @@ func init() {
 	kingpin.Flag("dbaas-metrics-path", "enable DO DBAAS metrics collection (this must be a DO DBAAS metrics endpoint)").
 		StringVar(&config.dbaas)
 
+	kingpin.Flag("mongodb-metrics-path", "enable mongodb metrics collection").
```

Comment:
Let's do `enable DO DBAAS mongodb metrics collection (this must be a DO DBAAS metrics endpoint)` like the previous argument to make it clear it is a flag for DO metrics.

Reply:
done